vizro-core/docs/index.md

```diff
@@ -1,9 +1,12 @@
 # Vizro
 
-Vizro is a toolkit for creating modular data visualization applications
+Vizro is a toolkit for creating modular data visualization applications.
+
+## First time user
```

Comment:
I would just take the headers ## First time user and ## Extensive documentation out and have a 5 card grid